cmakebuild.js

```diff
@@ -24,7 +24,7 @@ function runCmake (opts, target, cb) {
       }
     }
 
-    var proc = spawn(cmakeJsPath, args)
+    var proc = spawn(cmakeJsPath, args, { shell: true })
```

Comment:
I would adjust the PR to just add shell:true ob windows!

Reply:
> adjust the PR to just add shell:true ob windows!

Something like this?

```suggestion
    var proc = spawn(cmakeJsPath, args, process.platform === "win32" ? { shell: true } : {})
```

Reply:
I would go with undefined instead {}, but I would say yes

Reply:
```suggestion
    var proc = spawn(cmakeJsPath, args, process.platform === "win32" ? { shell: true } : undefined)
```

Reply:
LGTM from my PoV